.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -1,39 +1,63 @@
 name: Bug Report
-description: Something is not working right
+description: "Something is not working right"
 labels: ["bug"]
 body:
-  - type: markdown
+   - type: checkboxes
+    id: no-duplicate-issues
     attributes:
-      value: |
-        Before opening a new issue, take a moment to search through the current open ones.
-
-        ---
+      label: "Before opening a new issue, take a moment to search through the current open ones."
+      description: "Search in the issues sections by clicking [HERE](https://github.com/hyprwm/hyprland/issues?q=)"
+      options:
+        - label: "I have checked and did not find a similar issue"
+          required: true
 
   - type: textarea
     id: ver
     attributes:
-      label: Hyprland Version
+      label: "Hyprland Version"
       description: "Paste here the output of `hyprctl version`. For hyprland after 0.34.0, paste `hyprctl systeminfo` instead."
-      value: "<details>
-        <summary>System/Version info</summary>
+      render: shell
+      placeholder: |
+        $ hyprctl systeminfo
```

Comment:
I don't want this in a code block. It was in a spoiler for a reason. It's long and clutters the issue.

Reply:
the spoiler did not show

Reply:
![swappy-20240101_204216](https://github.com/hyprwm/Hyprland/assets/62766066/2d13126d-b531-4c98-a643-0e58b19fec27)

Reply:
click preview idiot

Reply:
it displays the text by default, who would go out of their way to click preview wtf?

Reply:
nobody and its not the point. The point is when they post it it will be a spoiler and wont clutter the issue.